tutor/src/store/courses.ts

```diff
@@ -108,7 +108,7 @@ export const useLatestCoursePreview = (offeringId: string | number) => useSelect
 export const useDisplayedCourseOfferingIds = () => {
     const courseOfferingIds = useAllCourses().map(c => c.offering_id)
     const selectedIds = UiSettings.get('displayedOfferingIds') || []
-    return union(courseOfferingIds, selectedIds)
+    return union(selectedIds, courseOfferingIds)
```

Comment:
@nathanstitt I swapped these two because the courseOfferingIds was basically overriding the selectedIds from the UISettings.

Reply:
hm, it seems like something else must be going on then.  maybe some of the ids are string and others number?

the order that things are passed to `union`  should not matter, for instance this should both be the same
`union([2], [1, 2,3])` and `union([1, 2],[2,3])` should both return `[1,2,3]`

Reply:
oh wait, maybe you mean the order that they're returned in is inconsistent?  I can see that happening sine union doesn't make any representations about the order.

Reply:
yes, the selectedIds is in the order that the user changed. The courseOfferingIds is in the default order.